src/bash/readme.md

```diff
@@ -0,0 +1,2 @@
+We will put the bash files for each set of cases here.
```

Comment:
This subfolder contains the bash scripts

Later, when files are added, you will add in this readme a short description of each file and its purpose